rmf_building_map_tools/building_map_model_downloader/building_map_model_downloader.py

```diff
@@ -166,12 +148,13 @@ def download_models(
 
 def main():
     args = parser.parse_args()
+    print("The -f option of package building_map_mode_downloader has been deprecated. \
```

Comment:
This gets printed regardless if the flag `-f` was used, I would suggest a cleaner deprecation using `argparse.Action`, I found an example gist here, https://gist.github.com/alzix/4868d66115a55567d3b43b32742f6f4f

Reply:
Addressed in e6744abdc2e7cafa00d144f9ad72d415817e64c1. I extended the deprecation notice to the -m flag as well, which is redundant now.